core/components/atoms/_chip/index.tsx

```diff
@@ -7,14 +7,14 @@ import { BaseProps, extractBaseProps } from '@/utils/types';
 import { IconProps, TextProps } from '@/index.type';
 
 export interface GenericChipProps extends BaseProps {
-  label: string;
+  label: string | React.ReactNode;
```

Comment:
The type defined for **label** in the prop table should be `ReactText` or `ReactNode`. Currently, it only shows `ReactNode`.

Reply:
ReactNode allow all types of values, so its overriding the string type. Going to use ReactElement for more specific prop type